docs/release-notes.txt

```diff
@@ -29,7 +29,7 @@ Version 0.12.4
 
 - Add a topic guide explaining Determined's approach to :ref:`elastic-infra-index`.
 
-- Add a topic guide explaining the :ref:`native-api`.
+- Add a topic guide explaining the :ref:`model-definitions_native-api`.
```

Comment:
blocking: I assume find-and-replace gone awry? Don't need to update release notes :)

Reply:
Oh oops, duh. Makes sense!